src/components/Home.tsx

```diff
@@ -9,8 +10,12 @@ export default function Home() {
     const user = useSelector(getUsername);
     
     return (
-        <div>
-            {user ? <p>Welcome, {user.username}!</p> : <p>Please log in</p>}
+        <div className="main-container">
+            {user ? 
+                <div className="grid-wrapper">
+                    <p className="grid-title">Welcome, {user.username}!</p>
+                    <Books />
+                </div> : <p className="grid-title">Please log in</p>}
```

Comment:
Nit - this ternary is not very easy to read (especially with this kind of formatting/lack of new lines/indentation), maybe put stuff in variables and reverse the order (almost like an early bail)